### DISCUSSION TITLE
quale reverse proxy

### DISCUSSION THREAD

**Discussion by federoco** [indomus/forum#2481] [Networking]
Ciao a tutti,
vorrei chiedervi un consiglio circa i reverse proxy.
Ho seguito le vostre guide ed installato Caddy tramite Docker su Raspeberry Pi, dove gira anche un'istanza di home assistant.
Tutto bene.

Sulla stessa rete ho un NAS synology, che offre nativamente la funzione di reverse proxy e anche certificati gratuiti let's encrypt.

Secondo voi quale dei due sistemi è più vantaggioso usare? 
Avere il proxy sul nas mi pare più comodo per l'interfaccia grafica e per il fatto che andrei ad installarci, tramite Docker, anche altri servizi che passerebbero dal proxy, quindi potrei configurare tutto più facilmente.
Magari, però, ci sono inconvenienti che non vedo.
Grazie dell'aiuto.

**Comment by snakuzzo** [comment]
Puoi fare come ti pare...la scelta è solo tua, non c'è un "meglio" o "peggio".
Non sono queste cose a fare la differenza tra meglio e peggio.

**Reply by federoco** [reply]
grazie!
